### DISCUSSION TITLE
Hack Browser Data outside the target computer

### DISCUSSION THREAD

**Discussion by xnvmx** [moonD4rk/HackBrowserData#299] [Q&A] [answered]
What Chrome files do I need to copy to recover all data (passwords, cookies, history, bookmarks, etc.) outside the target computer?
It seems to me that only the folder from path "C:\Users\kciek\AppData\Local\Google\Chrome\User Data\Default" is not enough. Or I am wrong?

**Comment by moonD4rk** [comment, marked as answer]
Hello, there are several types of data in Chrome that are encrypted (password/cookies/credit card), and they are encrypted through the API of the operating system. Therefore, it is impossible to export them after copying the data files to different computers. However, other unencrypted data can be restored from backups.
You can find the directory of the data file by entering `chrome://version/` in the browser input box. The directory under `Profile Path` is your current Chrome user's data folder, which you can backup.

**Reply by radpopl** [reply]
So if I have my entire profile saved on a pendrive, this command cannot work to recover passwords from Edge?

`hack-browser-data-windows-64bit>hack-browser-data-windows-64bit.exe -b all -p "V:\Users\myaccount\AppData\Local\Microsoft\Edge\User Data\Default"`
